typescript/infra/config/environments/mainnet3/warp/configGetters/getEthereumSeiPumpBTCWarpConfig.ts

```diff
@@ -23,7 +23,7 @@ export const getEthereumSeiPumpBTCWarpConfig = async (
 ): Promise<ChainMap<HypTokenRouterConfig>> => {
   const ethereum: HypTokenRouterConfig = {
     ...routerConfig.ethereum,
-    ...getOwnerConfigForAddress(ethereumOwner),
```

Comment:
I'm removing `getOwnerConfigForAddress()` because this getter outputs an overrides.proxyAdmin **and** proxyAdmin object. 
https://github.com/hyperlane-xyz/hyperlane-registry/blob/83cd45dba92a56d0b418b5941bb88d4fe0b04c65/deployments/warp_routes/pumpBTCsei/ethereum-sei-deploy.yaml#L5-L9

cc @paulbalaji